### PR TITLE
Fixes toggle comment

### PR DESCRIPTION
**Pull request by l33tlinuxh4x0r**
I have found an issue with my first revision of code. It literally toggles comments. For example:

```
#with

#comment

#would

#swap
```
would become
```
#
with
#
comment
#
would
#
swap
```
I just cleaned up the code and implemented a fix.

### CODE REVIEW

**Comment by l33tlinuxh4x0r** [comment]
Sorry about all of the commits. I'm still learning how to use github via command line.

File: pyzo/core/editor.py [hunk 1]
```diff
@@ -677,50 +677,38 @@ def uncommentBlock(cursor):
...
-                return
 
-        # Apply this function to all blocks
-        self.doForSelectedBlocks(toggleComment)
+            self.doForSelectedBlocks(getBlocks)
+            commented = [item for item in text_block if item.startswith("# ")]
+            if len(commented) == len(text_block):
+                self.doForSelectedBlocks(uncommentBlock)
+            else:
+                self.doForSelectedBlocks(commentBlock)
```
Comment (almarklein) [inline pyzo/core/editor.py:709]: nice!

**Comment by almarklein** [comment]
> Sorry about all of the commits.

No problem; I just squash it!

**Comment by almarklein** [comment]
To explain more: Github has the option to squash-merge a PR, which means that the PR becomes a single commit on master. I've been using this in more projects now. It helps make the history very clear.

**Pull request merged**